ui/README.adoc

```diff
@@ -8,9 +8,21 @@ ifdef::env-github[]
 :table-caption!:
 endif::[]
 
+:toc:
+:toc-placement!:
 
 = Mobile Security Service UI
 
+ifdef::status[]
+.*Project health*
+image:https://circleci.com/gh/aerogear/mobile-security-service.svg?style=svg[Build Status (CircleCI), link=https://circleci.com/gh/aerogear/mobile-security-service]
+image:https://coveralls.io/repos/github/aerogear/mobile-security-service/badge.svg?branch=master[Coverage Status (Coveralls), link=https://coveralls.io/github/aerogear/mobile-security-service?branch=master]
```

Comment:
I am not sure if coveralls is only reporting Go test coverage or not. Do you know the answer?

Reply:
good question @davidffrench, its only go right now, but 64% on badge is inconsistent with 96% reported coverage on [coveralls](https://coveralls.io/github/aerogear/mobile-security-service?branch=master). Will investigate

Reply:
@davidffrench best to remove this right now (the code cov badge) as its not related to the UI I think, so will do so now

Reply:
Agreed